src/docs/product/sentry-basics/search/saved-searches.mdx

```diff
@@ -2,72 +2,51 @@
 title: Saved Searches
 sidebar_order: 20
 keywords: ["saved search", "premade searches"]
-description: "Learn more about recommended, pinned, and saved searches."
+description: "Learn more about default, recommended, and saved searches."
 ---
 
-In the "Saved Searches" tab of **Issues**, you can access recommended searches, pin searches that you find most useful, and take advantage of organization-wide saved searches.
+Sentry allows you to save searches you've made on the **Issues** page, either for yourself or for your entire organization. You can access these in the "Saved Searches" sidebar by clicking the "Saved Searches" button at the top right of the page.
```

Comment:
Feeling more strongly about this. I think we should avoid saying out of the gate for you or your entire org because I don't want to set up an expectation that's not going to apply to a lot of users.

```suggestion
Sentry allows you to save searches you've made on the **Issues** page. You can access these in the "Saved Searches" sidebar by clicking the "Saved Searches" button at the top right of the page.
```